Automattic/wp-calypso
# A4A: Implement Pressable overview page

This PR implements the Pressable overview page based on the latest Purchase flow design. **Note that purchasing a plan is currently unavailable as the backend is being worked on. You may be able to add Pressable plans in the shopping cart, but you won't be able to obtain a license if you proceed to checkout.**

**Selecting a plan**
<img width="1273" alt="Screenshot 2024-03-26 at 2 49 21 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/e9632e04-a460-40e9-99a3-0bf33b7c4104">
**Selecting 'More**'
<img width="1273" alt="Screenshot 2024-03-26 at 2 49 59 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/aaa7768c-b5b1-4ac9-93b6-11cd30aebc87">




Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/74

## Proposed Changes

* Update the Slider component to accept a null value. This is something I wasn't able to consider as a valid use case earlier.
* Implement the Pressable page based on the latest design. Since the Figma design doesn't give us full information about what each Pressable plan includes, I have to rely on the existing Pressable pricing page for this information. https://pressable.com/pricing/

## Testing Instructions

* Use the live link and go to `/marketplace/hosting/pressable`
* Confirm that the page is rendered properly based on the latest design.
* Confirm that the slider works as expected.
* Confirm that selecting a plan will add it to the Shopping cart.
* Confirm that the page is responsive.

## Pre-merge Checklist
